src/cli/cli-report.c

```diff
@@ -230,18 +230,18 @@ static int read_crash_report_field(const char *text, problem_data_t *report,
     char newvalue[length + 1];
     strncpy(newvalue, textfield, length);
     newvalue[length] = '\0';
-    strtrim(newvalue);
+    libreport_strtrim(newvalue);
```

Comment:
Note: this crap can be replaced with `g_strchug()` and `g_strchomp()` where needed.

Reply:
Part of the future “nuke strbuf” quest.